packages/ui/src/theme/tokens/components/button.ts

```diff
@@ -63,7 +63,8 @@ export type ButtonTokens<Output extends OutputVariantKey> =
     | 'borderWidth'
     | 'borderStyle'
     | 'borderRadius'
-    | 'color'
+    | 'color',
+    Output
```

Comment:
The 2 types of issue are this one, where we forgot to pass the `Output` generic down, and the one below where we just forgot to add a `?`